Sources/xcodeproj/Utils/BuildSettingsProvider.swift

```diff
@@ -45,8 +45,10 @@ public class BuildSettingsProvider {
         var buildSettings: [String: Any] = [:]
         if let platform = platform, platform == .iOS {
             buildSettings["SDKROOT"] = "iphoneos"
-            buildSettings["CODE_SIGN_IDENTITY"] = "iPhone Developer"
             buildSettings["LD_RUNPATH_SEARCH_PATHS"] = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks"
+            if let variant = variant {
+                buildSettings["CODE_SIGN_IDENTITY"] = variant == .debug ? "iPhone Developer" : "iPhone Distribution"
```

Comment:
Hey @dangthaison91, only one more little thing. The stance of XcodeProj when it comes to default values is staying as aligned as possible to Xcode's behavior. In the case of `CODE_SIGN_IDENTITY`, Xcode sets the value to `iPhone Developer` regardless of the build configuration.
```suggestion
                buildSettings["CODE_SIGN_IDENTITY"] = "iPhone Developer"
```